internal/build/container_ops.go

```diff
@@ -3,57 +3,143 @@ package build
 import (
 	"bytes"
 	"context"
+	"fmt"
 	"io"
+	"io/ioutil"
 	"os"
 	"runtime"
+	"strings"
 
 	"github.com/BurntSushi/toml"
 	"github.com/docker/docker/api/types"
+	dcontainer "github.com/docker/docker/api/types/container"
```

Comment:
Could we do this camelCase? Is that not best practice for imports?

Reply:
I believe that the convention for package names (and hence, aliases, it seems) is all lowercase.